src/main/java/org/folio/dew/batch/bursarfeesfines/BursarExportJobConfig.java

```diff
@@ -76,6 +93,7 @@ public Step exportStep(
     JobRepository jobRepository,
     PlatformTransactionManager transactionManager
   ) {
+    log.info("In exportStepRegular");
```

Comment:
feel free to leave logs in here, just make sure they'll provide enough context in a logfile and can be useful in hunting down errors later.

Something like this may be better:
```suggestion
    log.info("Starting regular (non-aggregate) bursar export flow");
```